js/foundation.reveal.js

```diff
@@ -278,6 +278,11 @@ class Reveal extends Plugin {
       $('html').addClass('is-reveal-open');
     }
 
+    function removeRevealOpenClasses() {
+
```

Comment:
empty line

Reply:
Will remove it then.

Reply:
`addRevealOpenClasses` also has an empty line after the function declaration.